bin/SleepRayTune.py

```diff
@@ -57,20 +57,20 @@ def SleepRayTune(config, cfg: DictConfig, num_epochs, num_gpu):
 
 @hydra.main(config_path="../config", config_name="config")
 # model=m2, dataset=m2, training=m2 in terminal
-def main(cfg: DictConfig, num_epochs=100, gpus_per_trial=1):
+def main(cfg: DictConfig, num_epochs=200, gpus_per_trial=1):
 
     logger = logging.getLogger()
 
     # restrict the number for cuda
-    os.environ["CUDA_VISIBLE_DEVICES"] = "1,4,3"
+    os.environ["CUDA_VISIBLE_DEVICES"] = "1,4,3,2"
```

Comment:
It's better to not to hard code the values of environment variables. They can be set in the shell that you use to launch the script, or in your bashrc file